airflow/jobs/scheduler_job.py

```diff
@@ -798,6 +802,11 @@ def _execute(self) -> None:
                 except Exception:
                     self.log.exception("Exception when executing DagFileProcessorAgent.end")
             self.log.info("Exited execute loop")
+        return None
+
+    @staticmethod
+    def get_job_type() -> str:
+        return "SchedulerJob"
```

Comment:
From I can tell, this can simply be a class variable?

Reply:
yes.

Reply:
Done.